source/building-your-own-javascript-components/index.html.md.erb

```diff
@@ -253,6 +253,189 @@ createAll(ProjectComponent, undefined, {
 })
 ```
 
+## Building configurable components
+The component you’re building may need some configuration to get the information it needs to:
```

Comment:
**nitpick** The 'to:' ends up all alone on its own line even on wide screen. Admitedly that's something that's going to happen at other screen sizes, but should we fix that when the content container is at its widest?

<img width="840" alt="Screenshot 2024-11-19 at 17 29 32" src="https://github.com/user-attachments/assets/315fe91d-4a61-4f8e-87df-702a9090f47f">